<launch>
    <include file="$(find icog_face_tracker)/launch/camera.launch"/>
    <node name="tracker_node" pkg="icog_face_tracker" type="tracker.py" output="screen">
        <param name="window" value="false"/>
    </node>
</launch>